04_removeFromArray/removeFromArray.spec.js

```diff
@@ -13,8 +13,8 @@ describe('removeFromArray', () => {
   test.skip('ignores non present values', () => {
     expect(removeFromArray([1, 2, 3, 4], 7, "tacos")).toEqual([1, 2, 3, 4]);
   });
-  test.skip('ignores non present values, but still works', () => {
-    expect(removeFromArray([1, 2, 3, 4], 7, 2)).toEqual([1, 3, 4]);
+  test('ignores non present values, but still works', () => {
+    expect(removeFromArray([1,2, 2, 3, 4], 7, 2)).toEqual([1, 3, 4]);
```

Comment:
Reverting unrelated file to original state
```suggestion
  test.skip('ignores non present values, but still works', () => {
    expect(removeFromArray([1, 2, 3, 4], 7, 2)).toEqual([1, 3, 4]);
```